compiler/src/dotty/tools/backend/jvm/CodeGen.scala

```diff
@@ -85,8 +85,7 @@ class CodeGen(val int: DottyBackendInterface, val primitives: DottyPrimitives)(
         case ex: InterruptedException => throw ex
         case ex: CompilationUnit.SuspendException => throw ex
         case ex: Throwable =>
-          ex.printStackTrace()
-          report.error(s"Error while emitting ${unit.source}\n${ex.getMessage}", NoSourcePosition)
+          report.error(s"Error while emitting ${unit.source}\n${ex.getMessage}", cd.sourcePos)
```

Comment:
Spurious change?

Reply:
Hum, but that's too coarse of a fix. Now for bugs (unexpected exception) inside the backend, we will only ever see a message without a stack trace at all. That is going to be disastrous for further debugging.